openupgrade_scripts/scripts/purchase_mrp/15.0.1.0/upgrade_analysis.txt

```diff
@@ -3,3 +3,4 @@
 ---XML records in module 'purchase_mrp'---
 NEW ir.model.access: purchase_mrp.access_mrp_bom_line_purchase_user
 NEW ir.model.access: purchase_mrp.access_mrp_bom_purchase_user
+# NOTHING TO DO
```

Comment:
This change is done in a copy of the file, `upgrade_analysis_work.txt`.

Reply:
change done